gym/wrappers/record_episode_statistics.py

```diff
@@ -8,35 +8,6 @@
 import gym
 
 
-def add_vector_episode_statistics(
```

Comment:
Why have you removed this function?

Reply:
The function is only used for RecordEpisodeStatistics, and it undoes part of the point of this commit. 

Either way, when we finish an episode, we need to extend our `info` dictionary with the episode statistics in array form. 

In the original implementation, this was repeatedly called in a loop to produce statistics arrays and their corresponding masks, then insert them into the info dictionary

In this implementation, we create the exact same results by just using numpy indexing and the boolean done masks we already have. No need for an additional function that is used nowhere else

Reply:
Ok, that makes sense